client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -70,12 +70,27 @@ const renderBusinessExtensionHelpText = ( values, isInstallingActivating ) => {
 
 	const installingJetpackOrWcShipping =
 		extensions.includes( 'jetpack' ) ||
-		extensions.includes( 'woocommerce-shipping' );
+		extensions.includes( 'woocommerce-services:shipping' );
 
 	const accountRequiredText = __(
 		'User accounts are required to use these features.',
 		'woocommerce-admin'
 	);
+
+	const isWCShippingAvailableToInstall = extensions.includes(
+		'woocommerce-services:shipping'
```

Comment:
Should this and also the logic for `installingJetpackOrWcShipping` include `woocommerce-services:tax`, given that also installs WooCommerce services and would require Jetpack I take it?
This would mean we want to be more specific in the copy below and use `WooCommerce Tax` if that is the only item being installed?
cc @pmcpinto

Reply:
I think that makes sense

Reply:
The modification sounds good to me.
Now I'm thinking about 2 different solutions for this and @louwie17 I would like to know your opinion about them.

**Option 1**
As we're getting a list of free extensions ([just with a key, no name](https://github.com/woocommerce/woocommerce-admin/blob/main/src-internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php#L250-L416)) we can map (hardcode) the keys with the names we want: WooCommerce Shipping, WooCommerce Tax, and Jetpack.

**Option 2**
Add the prop name [to WC.COM](https://github.com/Automattic/woocommerce.com/blob/trunk/plugins/wccom-plugins/obw-free-extensions/includes/v3/extensions-v3.php#L118-L156) and the fallback and use always the `name`.

I have the feeling that option 2 looks more robust but maybe it's such a big change for something that we won't use anymore.

Reply:
Scratch my last comment. I found an easy way to do this.